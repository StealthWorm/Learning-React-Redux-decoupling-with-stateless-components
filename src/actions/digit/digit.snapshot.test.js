import renderer from 'react-test-renderer';
import { digitClick } from './index';

describe('digitClick action', () => {
  it('should have the correct shape', () => {
    const digit = 5;
    const action = digitClick(digit);
    expect(action).toMatchSnapshot();
  });
});