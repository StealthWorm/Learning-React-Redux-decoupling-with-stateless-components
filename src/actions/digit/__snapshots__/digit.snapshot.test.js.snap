// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`digitClick action should have the correct shape 1`] = `
Object {
  "payload": Object {
    "CLICKED_DIGIT_KEY": 5,
  },
  "type": "DIGIT_CLICK_ACTION",
}
`;
